### PR TITLE
Update profile icon and add minicart icon to IconPack

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?
Update profile icon and add minicart icon according to design data
P.S. This PR is related to the following prs:

1. https://github.com/vtex-apps/minicart/pull/53/
2. https://github.com/vtex-apps/login/pull/78/

 

#### How should this be manually tested?

https://filled--storecomponents.myvtex.com/

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "vtex.search-result": "2.x",
     "vtex.category-menu": "1.x",
     "vtex.breadcrumb": "0.x",
-    "vtex.login": "1.x",
+    "vtex.login": "2.x",
```
Comment (ArthurSampaio) [inline manifest.json]: @brunojdo I had to update the login version  to use the dreamstore iconPack

Comment (luciannojunior) [reply manifest.json]: He said not to change it. Revert it... Will not work for now, but that's ok

Comment (ArthurSampaio) [reply manifest.json]: ok. doing it right now

Comment (ArthurSampaio) [reply manifest.json]: done.

**Pull request merged**